src/target/codegen.cc

```diff
@@ -324,6 +333,46 @@ runtime::Module PackImportsToLLVM(const runtime::Module& mod, bool system_lib,
   return (*codegen_f)(blob_byte_array, system_lib, llvm_target_string, c_symbol_prefix);
 }
 
+struct Deleter {  // deleter
+  explicit Deleter(std::string file_name) { this->file_name = file_name; }
+  void operator()(FILE* p) const {
+    fclose(p);
+    ICHECK(remove(file_name.c_str()) == 0)
+        << "remove temporary file (" << file_name << ") unsuccessfully";
+  }
+  std::string file_name;
+};
+
+std::string SerializeModuleToBase64(tvm::runtime::Module module) {
+  static const runtime::PackedFunc* f_to_str = runtime::Registry::Get("serialize_runtime_module");
+  ICHECK(f_to_str) << "IndexError: Cannot find the packed function "
+                      "`serialize_runtime_module` in the global registry";
+  return (*f_to_str)(module);
+}
+
+tvm::runtime::Module DeserializeModuleFromBase64(std::string state) {
```

Comment:
state=> base64

- For loading (back to IRModule), actually we do not want to export load it as so, instead, we would like to directly load back, this is something that have not been implemented but likely very related with the process  blob